ios/Enums.swift

```diff
@@ -1,10 +1,17 @@
 // Pin notification actions for RN Bridge
-enum PinNotification: String {
+enum PinAuthNotification: String {
     case open
     case close
-    case showError
+    case incorrectPin
 }
 
+enum PinCreateNotification: String {
```

Comment:
I wonder about the reason to divide PinNotification to two separate notifications?

Reply:
is it to distinguish open/close from auth and create?

Reply:
Yes, previously we had a setup where the pin notification had a property for the type (auth/create). And this just makes it so that we have different events for it so it's a little easier to set up different handlers for it. 

This is useful for the following situation.

The pin create event has a pinLength property. 
The pin auth event does not. 

So you want to handle these events differently and this makes that a bit more intuitive.

Reply:
ok, so there is also an option to distinguish it differently, e.g.: 
```
enum PinNotification: String {
     case open(auth: Bool)
     case close(auth: Bool)
...
}
```
this way you have one enum but open(auth: true) means open from auth, open(auth: false) means open from create.
which one you'll choose does not matter, it's more like personal preference.

Reply:
But this way there is no distinction between the following two cases 
```
    case incorrectPin
    case pinNotAllowed
```

These cases are not meant for both the create **and** auth notification

Reply:
so in this case I'd stick with your solution :)